SparkPost/matchbox
# UX-364 Replace Select with ListBox on design site

### What Changed

- Replace `Select` with `ListBox` in the `LiveCode` component on the design site

### How To Test or Verify

- Run the design site locally
- Verify `ListBox` is being used and functioning correctly on the component code examples 

### PR Checklist
